src/moin/help/welcome/users-Home.data

```diff
@@ -0,0 +1 @@
+users/Home page
```

Comment:
could we replace this with a wiki page describing what is the users namespace and what can be done here?  as a beginner moin user the users namespace is mysterious to me.

Reply:
yes, good idea

Reply:
submitted this idea as #1416